Tools/Scripts/webkitpy/w3c/test_importer.py

```diff
@@ -212,6 +212,14 @@ def __init__(self, port, test_paths, options):
     def do_import(self):
         if self.source_directory:
             source_path = str(Path(self.source_directory) / 'web-platform-tests')
+            if not Path(source_path).exists():
```

Comment:
Instead of repeatedly creating and destroying `pathlib.Path` objects, we should just be using the existing filesystem functionality of `webkitpy`, because otherwise this is going to fail because `Path` doesn't go via our mock file system (`webkitpy.common.system.filesystem_mock.MockFileSystem`), which is likely the cause of the test failures.

We also don't want to allow any file named `web-platform-tests`, we want it to be a directory (per the error message logged below!).

So we want something like:

```suggestion
            source_path = self.filesystem.join(self.source_directory, 'web-platform-tests')
            if not self.filesystem.isdir(source_path):
```

Reply:
haha. We discussed this with @marcoscaceres 
Marcos was right that someone would comment about this. :) My bad. 
I will change thanks. 

Just curiosity: Do you know why `self.filesystem` was created?